__tests__/pages/api/companies/[id]/archiveCompany.test.ts

```diff
@@ -0,0 +1,146 @@
+import { prisma } from '@/lib/prisma'
```

Comment:
Please pay attention to a naming convention in tests
<img width="294" alt="Captura de Tela 2023-05-06 às 1 49 36 PM" src="https://user-images.githubusercontent.com/335095/236616973-a111436b-1ee9-44fc-b25d-bf5c38834329.png">
<img width="319" alt="Captura de Tela 2023-05-06 às 1 49 52 PM" src="https://user-images.githubusercontent.com/335095/236616975-fa004ab0-4d26-4321-91d1-d30e2db044fd.png">